src/session_impl.cpp

```diff
@@ -5661,7 +5655,7 @@ namespace aux {
 		m_dht_settings = settings;
 	}
 
-	void session_impl::set_dht_state(dht::dht_state const& state)
+	void session_impl::set_dht_state(dht::dht_state state)
 	{
 		m_dht_state = state;
```

Comment:
as far as I know, the only place compilers do that is in a ``return`` statement.